ctapipe/containers.py

```diff
@@ -487,17 +487,19 @@ class TriggerContainer(Container):
     container_prefix = ""
     time = Field(NAN_TIME, "central average time stamp")
     tels_with_trigger = Field(
-        [], "List of telescope ids that triggered the array event"
+        None, "List of telescope ids that triggered the array event"
     )
     event_type = Field(EventType.SUBARRAY, "Event type")
     tel = Field(Map(TelescopeTriggerContainer), "telescope-wise trigger information")
 
 
-class ReconstructedShowerContainer(Container):
+class ReconstructedGeometryContainer(Container):
```

Comment:
you should update the one in "__all__" otherwise it won't be read

Reply:
this also the reason why the docs fail

Reply:
done

Reply:
Is `Geometry` a good word here? More more commong would be `Direction`, right?

Reply:
well in principle it's not just the direction, but also the height of shower max, and potentially  (depending on the reco method) other aspects of the shower's geometry like the longitudinal and lateral widths,  Moliere radius, etc. a